core/src/epicli/data/common/ansible/playbooks/roles/kubernetes_master/files/coredns-configmap.yml

```diff
@@ -1,11 +1,13 @@
 # Based on https://github.com/kubernetes/kubernetes/blob/v1.18.6/cluster/addons/dns/coredns/coredns.yaml.in
-# with hosts plugin added
+# Hosts plugin added
 
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: coredns
   namespace: kube-system
+  labels:
+      addonmanager.kubernetes.io/mode: Reconcile
```

Comment:
```suggestion
      addonmanager.kubernetes.io/mode: EnsureExists
```

Reply:
I would like to still have the Reconcile value. Thanks to that we will stick to original state of CoreDNS pod in case of any unnecessary changes. Description already updated of the change.